src/s1reader/s1_reader.py

```diff
@@ -640,17 +720,21 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
         tree = ET.parse(f)
 
         product_info_element = tree.find('generalAnnotation/productInformation')
-        azimuth_steer_rate = np.radians(float(product_info_element.find('azimuthSteeringRate').text))
+        azimuth_steer_rate =\
+            np.radians(float(product_info_element.find('azimuthSteeringRate').text))
         radar_freq = float(product_info_element.find('radarFrequency').text)
         range_sampling_rate = float(product_info_element.find('rangeSamplingRate').text)
         orbit_direction = product_info_element.find('pass').text
 
         image_info_element = tree.find('imageAnnotation/imageInformation')
         azimuth_time_interval = float(image_info_element.find('azimuthTimeInterval').text)
         slant_range_time = float(image_info_element.find('slantRangeTime').text)
-        ascending_node_time = as_datetime(image_info_element.find('ascendingNodeTime').text)
+        ascending_node_time_annotation =\
+            as_datetime(image_info_element.find('ascendingNodeTime').text)
+        first_line_utc_time = as_datetime(image_info_element.find('productFirstLineUtcTime').text)
 
-        downlink_element = tree.find('generalAnnotation/downlinkInformationList/downlinkInformation')
+        downlink_element = tree.find('generalAnnotation/downlinkInformationList/'
+                                     'downlinkInformation')
```

Comment:
I totally agree with your point about the formatting consistency, especially in the level of whole codes in this repo. Therefore I think another issue / PR would be necessary.

In case of this PR, I wanted to keep the lines in reasonable length suggested by PEP8.